src/modules/atommodel/particles/tokens/RadixOwnedTokensParticle.ts

```diff
@@ -24,15 +26,28 @@ export class RadixOwnedTokensParticle extends RadixParticle {
     // tslint:disable-next-line:variable-name
     public token_reference: RadixResourceIdentifier
 
-    constructor(amount: BN, type: RadixFungibleType, address: RadixAddress, nonce: number, 
-                tokenReference: RadixTokenClassReference, planck?: number,
+    @includeDSON
+    @includeJSON
+    public granularity: RadixUInt256
+
+    constructor(
+        amount: BN,
+        granularity: RadixUInt256,
```

Comment:
I know this is isn't the point of this merge, but I think we should keep it consistent with the amount and use BN outside of partciles instead of UInt256

Reply:
Perfect, I've changed this. Thanks for the suggestion!